app/views/index/global.phtml

```diff
@@ -8,24 +8,23 @@
 		$class = ' state_unread';
 	}
 ?>
+<main id="stream" class="global<?= $class ?>">
+	<?php
```

Comment:
@mathContao This one's also a whitespace change, but pinging @Alkarex @aledeg as to whether it might be preferred.

Reply:
I do not understand here the issue

Reply:
Please look at the full diff, without ignoring whitespace. It's just a lot of lines that don't have anything to do with any of this.

Reply:
please help me. I do not understand the issue. I did not changed a lot on this files. Now it feels that I have to clean up a lot.

Reply:
Should be good now. But you did change those lines. ;-)